tutorials/algorithms/10_pvqd.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>Should we expect users to be familiar with Trotterization? I'd include at least a link to a page that explains it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yes, since this is a short tutorial I think we expect users to know (or look up) the basics. The PauliEvolutionGate has some more details and the link to a paper, I added that link </p>